where-to-travel/src/main/webapp/info.txt

```diff
@@ -0,0 +1,9 @@
+<h2 style="text-align: left; color: #049688;">Where To?</h2>
+<p style="text-align: center;"><strong>Planning a road trip? Enter the amount of time you want to spend on the road and we&rsquo;ll find interesting attractions for you!</strong></p>
+<h3 style="color: #049688;">Using Where To</h3>
```

Comment:
Add styling to CSS file. I know I have this same problem in some of my html, I will address it in my review.

Reply:
fixed!